tests/unit/test_hnswlib.cpp

```diff
@@ -57,32 +57,34 @@ TEST_F(HNSWLibTest, resizeNAlignIndex) {
 
     float a[dim];
 
-    // add up to n
+    // Add up to n.
     for (size_t i = 0; i < n; i++) {
         for (size_t j = 0; j < dim; j++) {
             a[j] = (float)i;
         }
         VecSimIndex_AddVector(index, (const void *)a, i);
     }
-    // the size and the capacity should be equal
+    // The size and the capacity should be equal.
     ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexCapacity(),
               VecSimIndex_IndexSize(index));
-    // the capcity shouldnt be changed
+    // The capcity shouldnt be changed.
     ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexCapacity(), n);
 
-    // add another vector to exceed the initial capacity
+    // Add another vector to exceed the initial capacity.
     VecSimIndex_AddVector(index, (const void *)a, n);
 
-    // the capacity should be now aligned with the block size
+    // The capacity should be now aligned with the block size.
+    // bs = 3, size = 11 -> capacity = 12
     ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexCapacity() % bs, 0);
-    // new capacity = intiailcapacity + blockSize - intiailcapacity % blockSize
+    // New capacity = intiailcapacity + blockSize - intiailcapacity % blockSize.
     ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexCapacity(),
               n + bs - n % bs);
-
+    // Check actual numbers **cahnge this if you change the intial parameters**
+    ASSERT_EQ(reinterpret_cast<HNSWIndex *>(index)->getHNSWIndex()->getIndexCapacity(), 12);
     VecSimIndex_Free(index);
 }
 
-// case 1: intiailCapacity much larger than block size, and it is not aligned
+// Case 1: intiailCapacity much larger than block size, and it is not aligned.
 TEST_F(HNSWLibTest, resizeNAlignIndex_initailCapacity_is_big) {
```

Comment:
Typo: initial capacity. Also, in comments I recommend writing English words rather than variable names as much as possible...